upgrade.php

```diff
@@ -244,7 +244,7 @@ function getSqlServer()
     //test if mysql or mariadb is used.
     $sql = 'SELECT VERSION() as version';
     $result = $link->query($sql);
-    if (strpos($result->fetch_array['version'], 'MariaDB') === false) {
+    if (strpos($result->fetch_array()['version'], 'MariaDB') === false) {
```

Comment:
Will it work with `$result->fetch_array()[0]['version']` or what is the problem here?